Deployment/Botazuredeploy.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
What are these 2 ARM templates, and how are they supposed to be used?

Reply:
We are using hybrid deployment here where O365 users and Microsoft Teams service are in GCC commercial tenant and other resources are created in GCC-H subscription. So, first ARM template only creates bot services in GCC commercial subscription and second ARM template creates other resources in GCC-H subscription.

Reply:
I see, can you move these to a gcc subfolder then?

Reply:
Moved GCC ARM templates to GCC subfolder.